cl/donate/api_views.py

```diff
@@ -196,11 +201,17 @@ def _get_membership_data(
                 "membershipName": data["membershipLevel"]["name"],
             }
 
+        payment_status = (
+            data.get("payments")[0].get("paymentStatus", "").lower()
```

Comment:
Here is a type hinting conflict: Unresolved attribute reference 'get' for class 'str'.

I think we need to update the type hint for `webhook_data`.